modules/telco-core-sr-iov.adoc

```diff
@@ -0,0 +1,39 @@
+// Module included in the following assemblies:
+//
+// * scalability_and_performance/telco_core_ref_design_specs/telco-core-rds.adoc
+
+:_mod-docs-content-type: REFERENCE
+[id="telco-core-sr-iov_{context}"]
+= SR-IOV
+
+New in this release::
+// https://issues.redhat.com/browse/CNF-12678
+* You can now create virtual functions for Mellanox NICs with the SR-IOV Network Operator when secure boot is enabled in the cluster host.
+Before you can create the virtual functions, you must first skip the firmware configuration for the Mellanox NIC and manually allocate the number of virtual functions in the firmware before switching the system to secure boot.
+
+Description::
+SR-IOV enables physical network interfaces (PFs) to be divided into multiple virtual functions (VFs).
```

Comment:
(PFs) is the proper abbreviation for "physical network interfaces" ? I see "physical functions" (PFs) in our docs too. I see both, just calling it out.